manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": 2,
+  "name": "emoji-everywhere",
+  "description": "Convert unicode emoji to images on all webpages",
+  "version": "0.1.5-beta",
+  "author": "Pierre Rudloff",
+  "developer": {
+    "name": "Pierre Rudloff <contact@rudloff.pro>"
+  },
+  "incognito": "spanning",
+  "bugs": "https://github.com/Rudloff/emoji-everywhere/issues",
+  "homepage_url": "https://addons.mozilla.org/firefox/addon/emoji-everywhere/",
+  "devDependencies": {
+    "grunt": "~1.0.0",
+    "grunt-contrib-csslint": "~1.0.0",
+    "grunt-jslint": "~1.1.14",
+    "grunt-jsonlint": "~1.1.0"
+  },
+  "keywords": [
+    "emoji",
+    "firefox",
+    "twemoji"
+  ],
+  "icons": {
+    "48": "icons/emoji-48.png",
+    "96": "icons/emoji-96.png"
+  },
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
+      "js": [
+	    "data/js/twemoji.min.js",
+	    "data/js/emoji.js"
+	  ],
+	  "css": ["data/css/emoji.css"]
+    }
+  ],
```

Comment:
We will also need something like this in order to allow the add-on to load the images:
```json
"web_accessible_resources": [
  "data/components/twemoji/2/72x72/*.png"
],
```